dashboard/legacy/middleware/files_api.rb

```diff
@@ -23,6 +23,8 @@ def max_app_size
   # Can set this to an empty array if we do not want aichat checked for profanity.
   LABS_TO_CHECK_FOR_PROFANITY = DCDO.get('labs_to_check_for_profanity', [])
 
+  BACKPACK_PROGRAM_FILE_TYPES = ['.java', '.py']
```

Comment:
should we also add csv and txt here?

Reply:
Yeah, I wasn't sure about this. That means that if `.csv` and `.txt` files in other labs are not checked for profanity for the 'libraries' endpoint. I know that Applab uses the libraries endpoint for their libraries feature so that the shared .js code is moderated. I'm going to do a little more digging into this. Maybe we can decide in a follow-up?

Reply:
yep fine to look into as a follow-up! Better to over-check for profanity